kaldb/src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -75,7 +75,8 @@ public static void main(String[] args) throws Exception {
     }
     Path configFilePath = Path.of(args[0]);
 
-    Kaldb kalDb = new Kaldb(configFilePath);
+    KaldbConfig.initFromFile(configFilePath);
+    Kaldb kalDb = new Kaldb(KaldbConfig.get());
     kalDb.start();
```

Comment:
I think we want `start` to take `KaldbConfig` as an argument so that we can potentially reuse the `start` method in the test cases

Reply:
Sure, we can make that change when we change the tests. For now the tests work well.